File: chatwoot/ruby/src/GetDetailsOfAllIntegrationsExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    response = ChatwootClient::IntegrationsApi.new.get_details_of_all_integrations(
        0, # account_id
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling IntegrationsApi#get_details_of_all_integrations: #{e}"
end

File: chatwoot/python/src/GetDetailsOfAllIntegrationsExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
    # api_key={"agentBotApiKey": "AGENT_BOT_API_KEY"},
    # api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.IntegrationsApi(api_client).get_details_of_all_integrations(
            account_id=0,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling IntegrationsApi#get_details_of_all_integrations: %s\n" % e)
